src/components/personkort/hooks/useGetVeilederBrukerKnytning.tsx

```diff
@@ -0,0 +1,26 @@
+import { useValgtPersonident } from "@/hooks/useValgtBruker";
+import { get } from "@/api/axios";
+import { useQuery } from "@tanstack/react-query";
+import { SYFOOVERSIKTSRV_PERSONTILDELING_ROOT } from "@/apiConstants";
+
+export function useGetVeilederBrukerKnytning():
+  | VeilederBrukerKnytningDTO
+  | undefined {
+  const personident = useValgtPersonident();
+  const path = `${SYFOOVERSIKTSRV_PERSONTILDELING_ROOT}/personer/single`;
+  const getVeilederBrukerKnytning = () =>
+    get<VeilederBrukerKnytningDTO>(path, personident);
+  const { data: veilederBrukerKnytning } = useQuery({
+    queryKey: ["veilederBrukerKnytning", personident],
+    queryFn: getVeilederBrukerKnytning,
+    enabled: !!personident,
+  });
+
+  return veilederBrukerKnytning;
+}
+
+export interface VeilederBrukerKnytningDTO {
+  personident: string;
+  tildeltVeilederident?: string;
+  tildeltEnhet?: string;
```

Comment:
Burde disse også kunne representeres som `null` 🤔 En `null` verdi vil jo fortelle oss at det ikke er tildelt noe, imens fraværet av verdien kanskje er mer intetsigende
```suggestion
  tildeltVeilederident?: string | null;
  tildeltEnhet?: string | null;
```

Reply:
Tror det egentlig bør være `tildeltEnhet: string | null` og tilsvarende. Feltet er ikke optional, men kan ha null-verdi.